teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md

```diff
@@ -18,7 +18,7 @@ This cmdlet creates a Shifts connection instance.
 ## SYNTAX
 
 ```
-New-CsTeamsShiftsConnectionInstance -ConnectorId <string> -ConnectorSpecificSettingAdminApiUrl <string> -ConnectorSpecificSettingCookieAuthUrl <string> -ConnectorSpecificSettingEssApiUrl <string> -ConnectorSpecificSettingFederatedAuthUrl <string> -ConnectorSpecificSettingLoginPwd <string> -ConnectorSpecificSettingLoginUserName <string> -ConnectorSpecificSettingRetailWebApiUrl <string> -ConnectorSpecificSettingSiteManagerUrl <string> -Name <string> -DesignatedActorId <string> -EnabledConnectorScenario <string[]> -EnabledWfiScenario <string[]> -SyncFrequencyInMin <Integer> [-ConnectorAdminEmail <String[]>] [<CommonParameters>]
+New-CsTeamsShiftsConnectionInstance -ConnectorId <string> -ConnectorSpecificSettings <Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.IConnectorSpecificUkgDimensionsSettingsRequest> -Name <string> -DesignatedActorId <string> -EnabledConnectorScenario <string[]> -EnabledWfiScenario <string[]> -SyncFrequencyInMin <Integer> [-ConnectorAdminEmail <String[]>] [<CommonParameters>]
```

Comment:
I don't understand this update. I can see the parameters as they were in the documentation in TPM v 4.6.0.
Is there an update in the next version?

Reply:
as the new version of the service now supports 2 different connectors that require two different sets of parameters, the cmdlets (New and Set) now accept an object as input for the single parameter "ConnectorSpecificSettings" rather than all the single values on the command line. Common parameters stay as they were. Does this answer your question?

Reply:
Yes, but in that case what the article needs is 2 parameter sets in the syntax section. This is what I see with PlatyPS:

## SYNTAX

### NewExpanded (Default)
```
New-CsTeamsShiftsConnectionInstance -ConnectorId <String> -ConnectorSpecificSettingAdminApiUrl <String>
 -ConnectorSpecificSettingCookieAuthUrl <String> -ConnectorSpecificSettingEssApiUrl <String>
 -ConnectorSpecificSettingFederatedAuthUrl <String> -ConnectorSpecificSettingLoginPwd <String>
 -ConnectorSpecificSettingLoginUserName <String> -ConnectorSpecificSettingRetailWebApiUrl <String>
 -ConnectorSpecificSettingSiteManagerUrl <String> -DesignatedActorId <String>
 -EnabledConnectorScenario <String[]> -EnabledWfiScenario <String[]> -Name <String> -SyncFrequencyInMin <Int32>
 [-ConnectorAdminEmail <String[]>] [-WorkforceIntegrationId <String>] [-Break]
 [-HttpPipelineAppend <SendAsyncStep[]>] [-HttpPipelinePrepend <SendAsyncStep[]>] [-Proxy <Uri>]
 [-ProxyCredential <PSCredential>] [-ProxyUseDefaultCredentials] [-WhatIf] [-Confirm] [<CommonParameters>]
```

### New
```
New-CsTeamsShiftsConnectionInstance -Body <IConnectorInstanceRequest> [-Break]
 [-HttpPipelineAppend <SendAsyncStep[]>] [-HttpPipelinePrepend <SendAsyncStep[]>] [-Proxy <Uri>]
 [-ProxyCredential <PSCredential>] [-ProxyUseDefaultCredentials] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Reply:
How to use PlatyPS:
https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/repo_docs/NEW_CMDLETS.md